src/simulation.py

```diff
@@ -8,14 +8,15 @@
 class Simulation():
     def __init__(self):
         self.config = Config()
+        self.generation_duration = self.config.generation_duration
         self.world = World(self.config)
         self.population = Population(self.config)
         self.screen = screen_init(self.config)
         self.running = True
     
     def run(self):
         pygame.init()
```

Comment:
I think self.running should be set as False in the constructor and only be true if self.run() is called.
```suggestion
        self.running = False
    
    def run(self):
        self.running = True
        pygame.init()
```